examples/librascal/README.md

```diff
@@ -0,0 +1 @@
+Should get the model from librascal or somewhere. TBD.
```

Comment:
er, let's expand this... how about using the Zundel example from librascal?

Reply:
see latest commit.